py3status/modules/netdata.py

```diff
@@ -24,6 +24,12 @@
     {total}    color threshold of total usage
 
 @author Shahin Azad <ishahinism at Gmail>
+
+SAMPLE OUTPUT
+[
+    {'color': '#FF0000', 'full_text': 'LAN(Kb):  16.9↓   5.2↑', 'separator': True},
+    {'color': '#FF0000', 'full_text': 'T(Mb): 2264↓ 143↑ 2407↕'}
```

Comment:
No more separator. We should use different color too. Throw `{nic}` in there too. I think this...
```
[
{'full_text': 'eno1'},
{'full_text': 'LAN(Kb): 671.9↓  31.4↑ ', 'color': '#00ff00'},
{'full_text': 'T(Mb): 14130↓ 702↑ 14832↕', 'color': '#ff0000'},
]
```